api/v1alpha1/osbuild_types.go

```diff
@@ -90,7 +81,17 @@ type OSBuildCondition struct {
 	LastTransitionTime *metav1.Time `json:"lastTransitionTime,omitempty" description:"last time the condition transit from one status to another"`
 }
 
-type OSBuildConditionType string
+type ConditionType string
+
+// These are the valid OSBuild condition types
+const (
+	// Whether OSBuild build is done and ready
+	ConditionReady ConditionType = "Ready"
+	// Whether OSBuild build is in progress
+	ConditionProgressing ConditionType = "Progressing"
```

Comment:
Please change `Progressing` to `InProgress` (both const name and content)

Reply:
Done